README.md

```diff
@@ -1335,7 +1335,7 @@ It supports the following Spark deploy modes:
 
 ### Configuring Spark Project
 
-When using Spark backend uncomment the following Spark configuration in the application.conf file
+When using Spark backend uncomment the following Spark configuration in the reference.conf file, which is available under `core/src/resources`
```

Comment:
I'm not 100% sure how these work, but I don't think `reference.conf` is ever read directly. Instead we'd want to copy the spark section across to `application.conf`?

Happy to be told otherwise...